let a = "hello"
try {
    let no = parseInt(aa)
    console.log(no)
} catch (e) {
    console.log("error occurred")
    console.log(e.name)
    console.log(e.message) // 
}